lib/string_reverse.rb

```diff
@@ -17,3 +17,9 @@ def string_reverse(my_string)
 
   return my_string
 end
+

```

Comment:
Technically, since you're reversing the string in place, you don't need to return nil or my_string and lines 4 and 18 can simply be `return`